conftest.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+from users.models import User, Team, Role
+from tasks.models import Task, Priority, Status
+from falcon.test_support import Random
+
+
+@pytest.fixture
+def teams():
+    """
+    Adds teams to DB
+    """
+    team1 = Team.objects.create(name="Team1",
+                                description="This is a test team")
+    team2 = Team.objects.create(name="Team2",
+                                description="This is a test team")
+    team3 = Team.objects.create(name="Team3",
+                                description="This is a test team")
+    return team1, team2, team3
+
+
+@pytest.fixture
+def users(teams):
+    """
+    Adds users to DB
+    """
+    team1, team2, team3 = teams
+    employees = []
+    managers = []
+    users_counter = 0
+    for i in range(3):
+        for team in (team1, team2):
+            print()
+            employee = User.create_user(f"User{users_counter}", Random.email(), "ssSSAD231!@",
+                                        Random.string(5), Random.string(5), Role.EMPLOYEE, team)
```

Comment:
I wouldn't use random input in unit test. Random test are important, but unit test should be more robust and predictable. This is the first layer of code reliability and I think that random tests are fit for this purpose.